ostree-backup.sh

```diff
@@ -63,7 +63,8 @@ else
 fi
 
 if [[ ! -f /var/tmp/backup/etc.tgz ]]; then
-    ostree admin config-diff | awk '{print "/etc/" $2}' | xargs tar czf /var/tmp/backup/etc.tgz --selinux
+    # TODO: We're currently ignoring the deletion of these files, which means in the new cluster those files are gonna going to be present. We need to figure out a way how to also delete them on the new stateroot
+    ostree admin config-diff | awk '!/^D/ {print "/etc/" $2}' | xargs tar czf /var/tmp/backup/etc.tgz --selinux
```

Comment:
The deletions can be recorded in a similar fashion:
`ostree admin config-diff | awk '/^D/ {print "/etc/" $2}' > /var/tmp/backup/etc.deletions`

In the ostree-restore.sh, then, we can do:
`ostree cat $backup_tag /etc.deletions | xargs --no-run-if-empty -ifile rm -f /ostree/deploy/$new_osname/deploy/$ostree_deploy/file`

Reply:
Thanks, Don. Updated according to your suggestions. Please review.